functions/ecom.config.js

```diff
@@ -6,12 +6,12 @@
  */
 
 const app = {
-  app_id: 124951,
-  title: 'My Awesome E-Com Plus App',
-  slug: 'my-awesome-app',
+  app_id: 124679,
+  title: 'Intelipost',
+  slug: 'intelipost',
   type: 'external',
   state: 'active',
-  authentication: true,
+  authentication: false,
```

Comment:
```suggestion
  authentication: true,
```

> Isso, foi já pensando no tracking

Então precisa de autenticação...